### PR TITLE
packaging: Fix missing commit message in building kata-runtime

### PR DESCRIPTION
**Pull request by Bevisy**
add `git` package to the shim-v2 build image

Fixes: #3196

Signed-off-by: Binbin Zhang <binbin36520@gmail.com>

### CODE REVIEW

File: tools/packaging/static-build/shim-v2/Dockerfile [hunk 1]
```diff
@@ -2,10 +2,16 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu
+FROM ubuntu:20.04
```
Comment (fidencio) [inline tools/packaging/static-build/shim-v2/Dockerfile:5]: This is a valid change, but I'd like to ask you to split this into a different commit.  Also, please, check if the other Dockerfiles need the same change.

File: tools/packaging/static-build/shim-v2/Dockerfile [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && apt-get install -y make curl sudo gcc
+RUN apt-get update && \
+    apt-get install -y \
+        curl \
+        gcc \
+        git \
+        make \
+        sudo
```
Comment (fidencio) [inline tools/packaging/static-build/shim-v2/Dockerfile:14]: This part is good to go! :-)

**Review by fidencio — CHANGES_REQUESTED** [review]
@Bevisy, thanks for the PR!
I've added one comment inline, and once that's fixed we can have this one merged.

File: tools/packaging/static-build/shim-v2/Dockerfile (hunk 1 above)
Comment (Bevisy) [reply tools/packaging/static-build/shim-v2/Dockerfile:5]: Ok, I will remove this change. There is a nother pr #3109 which is used to delint dockerfile.

**Comment by Bevisy** [comment]
```sh
❯ ./kata-runtime --help
NAME:
   kata-runtime - kata-runtime runtime

kata-runtime is a command line program for running applications packaged
according to the Open Container Initiative (OCI).

USAGE:
   kata-runtime [global options] command [command options] [arguments...]

VERSION:
   kata-runtime  : 2.4.0-alpha0
   commit   : 31547cfbcb190bd20d228aea346fd7e996aac27d       <--- fixed
   OCI specs: 1.0.2-dev
```

**Review by fidencio — APPROVED** [review]
lgtm, thanks @Bevisy!

**Comment by fidencio** [comment]
@Bevisy, and about the other change you proposed (fixing a version of ubuntu in the dockerfiles), please, open a new PR for that as that's worth having fixed as well.

**Comment by Bevisy** [comment]
> 

By reviewing #3109, I found that this problem happened to be solved by adding `git`, and it also included the modification of the specific version of the image. So we may not open a new PR for the time being to fix the image version problem, but we still need to create a new backport PR to ensure that this modification can appear in release v2.3.

**Review by liubin — APPROVED** [review]
